tools/esp8266/app.cpp

```diff
@@ -411,33 +402,42 @@ void app::processPayload(bool retransmit) {
                                             snprintf(topic, 32 + MAX_NAME_LENGTH, "%s/ch%d/%s", iv->name, rec->assign[i].ch, fields[rec->assign[i].fieldId]);
                                             snprintf(val, 10, "%.3f", iv->getValue(i, rec));
                                             mMqtt.sendMsg(topic, val);
-                                            if(recRealtime == rec) {
-                                                if(CH0 == rec->assign[i].ch) {
-                                                    switch(rec->assign[i].fieldId) {
-                                                        case FLD_PAC: total[0] += iv->getValue(i, rec); break;
-                                                        case FLD_YT:  total[1] += iv->getValue(i, rec); break;
-                                                        case FLD_YD:  total[2] += iv->getValue(i, rec); break;
-                                                        case FLD_PDC: total[3] += iv->getValue(i, rec); break;
+                                            if (recRealtime == rec) {
+                                                if (CH0 == rec->assign[i].ch) {
+                                                    switch (rec->assign[i].fieldId) {
+                                                        case FLD_PAC:
+                                                            total[0] += iv->getValue(i, rec);
+                                                            break;
+                                                        case FLD_YT:
+                                                            total[1] += iv->getValue(i, rec);
+                                                            break;
+                                                        case FLD_YD:
+                                                            total[2] += iv->getValue(i, rec);
+                                                            break;
+                                                        case FLD_PDC:
+                                                            total[3] += iv->getValue(i, rec);
+                                                            break;
```

Comment:
ist schade, dass die kurze anweisung jetzt unnötig lang wird, aber Abstriche muss man wohl machen bei der automatischen Formatierung

Reply:
Ja gebe dir recht. 
Ich kann es mir mal auf die Todo Liste setzen, damit es wie vorher in einer Zeile ist. 
Bei solch kurzen Abfragen ist es schöner, da geb ich dir Recht.